src/onnx/onnx_parser.cpp

```diff
@@ -149,6 +149,23 @@ instruction_ref onnx_parser::node_info::add_broadcastable_binary_op(const std::s
     return this->add_common_op(op_name, arg0, arg1);
 }
 
+/**
+ * @brief A wrapper for insert_common_args(), which constructs an argument list
+ * and inserts multibroadcast ops to broadcast and typecast inputs to a common type and shape
+ * as required.
+ *
+ * Use add_common_op() to match input sizes when inputs may be
+ *  either static or dynamic.
+ *
+ * @param op_name               string; Name of operation (op) to add; valid names are the same as
+ * for make_op()
+ *
+ * @param inputs                vector of instruction_ref.  List of instructions which are the
+ * inputs to the new operator.
+ *
+ * @return instruction_ref      Return is an instruction_ref which can be used as input to any
+ * subsequent operation.
```

Comment:
I don't understand what this is trying to say. The output will be the result of the operation.

Reply:
Updated.  This stuff is confusing for anyone who doesn't already know it inside and out.